server/src/visitors/YmlFunctionVisitor.ts

```diff
@@ -28,7 +41,9 @@ export class YmlFunctionVisitor extends YmlBaseVisitor {
 
     private functionName: string;
 
-    private func: AbstractYmlObject;
+    private func: AbstractYmlFunction;
+
+    private previousOperatorIsAnd: boolean = undefined;
```

Comment:
Maybe add a comment explaining why this matters? xD

Reply:
Oh, yeah. Sorry.

Reply:
Oh OK so according to this comment, it could actually be “previousOperatorIsOr”; what matters is that you detect changes.